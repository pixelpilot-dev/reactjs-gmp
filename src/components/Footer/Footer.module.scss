.footer {
  background-color: $bg-dark;

  .container {
    padding-top: 23px;
    padding-bottom: 23px;
  }

  .logo {
    display: block;
    width: 152px;
    margin: auto;

    @include for-mobile {
      width: 100px;
    }
  }
}
