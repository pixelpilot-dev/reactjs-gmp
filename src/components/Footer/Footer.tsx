import * as React from 'react';
import { Link } from 'react-router-dom';
import { LOCALE, I18Y } from '../../core/i18y';
import { Wrapper } from '../Wrapper';
import logo from '../../assets/images/netflixroulette.svg';

import styles from './Footer.module.scss';

export const Footer = () => {
  return (
    <footer className={styles.footer}>
      <Wrapper className={styles.container}>
        <Link to='/' className={styles.logo}>
          <img src={logo} alt={I18Y[LOCALE].LOGOTYPE_ALT} />
        </Link>
      </Wrapper>
    </footer>
  );
};
