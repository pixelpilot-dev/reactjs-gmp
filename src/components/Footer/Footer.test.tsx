import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer component', () => {
  it('should be render correctly', () => {
    const { asFragment } = render(<Footer />, { wrapper: BrowserRouter });

    expect(asFragment()).toMatchSnapshot();
  });
});
