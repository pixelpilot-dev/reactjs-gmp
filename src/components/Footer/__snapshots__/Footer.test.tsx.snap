// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="wrapper container"
    >
      <a
        class="logo"
        href="/"
      >
        <img
          alt="NetflixRoulette"
          src="netflixroulette.svg"
        />
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
